src/Utility/TestMailer.php

```diff
@@ -27,6 +31,24 @@ public function __construct(
         $this->testSessionEnvironment = TestSessionEnvironment::singleton();
     }
 
+    public function send(RawMessage $message, Envelope $envelope = null): void
+    {
+        parent::send($message, $envelope);
+        if (!is_a($message, Email::class)) {
+            throw new InvalidArgumentException('$message must be a ' . Email::class);
+        }
```

Comment:
```suggestion
```
This is already checked and thrown in `parent::send()`

Reply:
Updated